### PR TITLE
Allow combmatrix.panel.line.size = 0

### LINKED ISSUE #12: Minor issue: thin lines remain with `combmatrix.panel.line.size = 0`
Hi @const-ae, 

I noticed that if I set `theme_combmatrix(combmatrix.panel.line.size = 0)` and save the plot as a PDF, I still get thin lines connecting the sets. This is super minor, I just thought I should note it in case others are confused about their PDF output.

Example:
```r
library(ggplot2)
library(ggupset)
library(dplyr)
plt <- tidy_movies %>%
  distinct(title, year, length, .keep_all=TRUE) %>%
  ggplot(aes(x=Genres)) +
    geom_bar() +
    scale_x_upset() +
    theme_combmatrix(combmatrix.panel.line.size = 0)
ggsave("test_plot.pdf", plot=plt)

```
[test_plot.pdf](https://github.com/const-ae/ggupset/files/4495553/test_plot.pdf)

### PR DESCRIPTION
**Pull request by karldw**
This change makes a special case for `combmatrix.panel.line.size == 0`. Let me know if I should make any changes!

Closes #12

### CODE REVIEW

File: R/axis_combmatrix.R [hunk 1]
```diff
@@ -254,10 +254,21 @@ make_combination_matrix_plot <- function(labels, labels_split, label_set, range,
   if(isTRUE(theme$combmatrix.panel.striped_background)){
     plt <- plt + geom_rect(aes(fill= .data$index %% 2 == 0), ymin=df2$index-0.5, ymax=df2$index+0.5, xmin=0, xmax=1)
   }
+  if (!isTRUE(theme$combmatrix.panel.line.size == 0)) {
```
Comment (const-ae) [inline R/axis_combmatrix.R:257]: This empty if statement could be deleted, right?
If yes, please do so :)

File: R/axis_combmatrix.R [hunk 2]
```diff
@@ -254,10 +254,21 @@ make_combination_matrix_plot <- function(labels, labels_split, label_set, range,
...
+  }
+
+  plt <- plt +
+    geom_point(aes(color= .data$observed), size=theme$combmatrix.panel.point.size)
+  if (!isTRUE(theme$combmatrix.panel.line.size == 0)) {
```
Comment (const-ae) [inline R/axis_combmatrix.R:262]: the `isTRUE()` is necessary to defend against `NA` or `NULL`, right?

Comment (karldw) [reply R/axis_combmatrix.R:262]: Yes, or `combmatrix.panel.line.size` with length > 1

File: R/axis_combmatrix.R (hunk 1 above)
Comment (karldw) [reply R/axis_combmatrix.R:257]: Yes! Sorry about that.

File: R/axis_combmatrix.R (hunk 2 above)
Comment (const-ae) [reply R/axis_combmatrix.R:260]: Oh, I didn't realize that it helps in that case too. Great :)

**Comment by const-ae** [comment]
I have merged the changes. Thank you again for your effort Karl :)

**Pull request merged**